mailing/src/test/java/org/lukawska/trainSmart/mailing/application/service/MailTestData.java

```diff
@@ -0,0 +1,62 @@
+package org.lukawska.trainSmart.mailing.application.service;
+
+import lombok.NoArgsConstructor;
+import org.lukawska.trainSmart.mailing.application.dto.MailResponse;
+import org.lukawska.trainSmart.mailing.domain.entities.MailEntity;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.stream.IntStream;
+
+@NoArgsConstructor(access = lombok.AccessLevel.PRIVATE)
+public final class MailTestData {
```

Comment:
czy sens jest miec ta klase publiczna tak samo te metody? pytam bo nie chce mi sie analizowac pakietow. Ale jezeli uzywasz to tylko w obrebie jednego pakietu i ten modul jest skonczony to nie ma sensu tak miec